hummingbird/ml/operator_converters/sklearn/sv.py

```diff
@@ -52,7 +54,7 @@ def forward(self, x):
             # using quadratic expansion--susseptible to rounding-off errors
             # http://www.robots.ox.ac.uk/~albanie/notes/Euclidean_distance_trick.pdf
             x_norm = -self.gamma * (x ** 2).sum(1).view(-1, 1)
-            k = torch.exp(x_norm + self.sv_norm + 2.0 * self.gamma * torch.mm(x, self.sv_t))
+            k = torch.exp(x_norm + self.sv_norm + 2.0 * self.gamma * torch.mm(x, self.sv_t).double())
```

Comment:
was this a bug? or why this change?

Reply:
This is because TVM wants the type right.